[2025-08-18 23:53:02,272] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: example_branch_dop_operator_v3.condition scheduled__2025-08-19T02:52:00+00:00 [queued]>
[2025-08-18 23:53:02,280] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: example_branch_dop_operator_v3.condition scheduled__2025-08-19T02:52:00+00:00 [queued]>
[2025-08-18 23:53:02,280] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2025-08-18 23:53:02,281] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2025-08-18 23:53:02,281] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2025-08-18 23:53:02,298] {taskinstance.py:1377} INFO - Executing <Task(BranchPythonOperator): condition> on 2025-08-19 02:52:00+00:00
[2025-08-18 23:53:02,300] {standard_task_runner.py:52} INFO - Started process 2231840 to run task
[2025-08-18 23:53:02,305] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'example_branch_dop_operator_v3', 'condition', 'scheduled__2025-08-19T02:52:00+00:00', '--job-id', '358', '--raw', '--subdir', '/root/Documents/airflow/venv/lib64/python3.9/site-packages/airflow/example_dags/example_branch_python_dop_operator_3.py', '--cfg-path', '/tmp/tmpdsd7gk2e', '--error-file', '/tmp/tmpu9b7ef51']
[2025-08-18 23:53:02,305] {standard_task_runner.py:80} INFO - Job 358: Subtask condition
[2025-08-18 23:53:02,346] {task_command.py:370} INFO - Running <TaskInstance: example_branch_dop_operator_v3.condition scheduled__2025-08-19T02:52:00+00:00 [running]> on host itoneptl
[2025-08-18 23:53:02,403] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_branch_dop_operator_v3
AIRFLOW_CTX_TASK_ID=condition
AIRFLOW_CTX_EXECUTION_DATE=2025-08-19T02:52:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-08-19T02:52:00+00:00
[2025-08-18 23:53:02,410] {logging_mixin.py:115} WARNING - /root/Documents/airflow/venv/lib64/python3.9/site-packages/airflow/utils/context.py:290 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2025-08-18 23:53:02,410] {logging_mixin.py:115} INFO - ------------- exec dttm = 2025-08-19T02:52:00+00:00 and minute = 52
[2025-08-18 23:53:02,411] {python.py:173} INFO - Done. Returned value was: empty_task_1
[2025-08-18 23:53:02,411] {skipmixin.py:140} INFO - Following branch empty_task_1
[2025-08-18 23:53:02,411] {skipmixin.py:175} INFO - Skipping tasks ['empty_task_2']
[2025-08-18 23:53:02,442] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=example_branch_dop_operator_v3, task_id=condition, execution_date=20250819T025200, start_date=20250819T025302, end_date=20250819T025302
[2025-08-18 23:53:02,476] {local_task_job.py:156} INFO - Task exited with return code 0
[2025-08-18 23:53:02,505] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
